buildpacktoml_test.go

```diff
@@ -286,32 +288,107 @@ func testBuildpackTOML(t *testing.T, when spec.G, it spec.S) {
 					}
 				})
 
-				it("should include processes", func() {
-					h.Mkfile(t,
-						`[[processes]]`+"\n"+
-							`type = "some-type"`+"\n"+
-							`command = "some-cmd"`+"\n"+
+				when("processes", func() {
+					when("platform API < 0.6", func() {
+						it.Before(func() {
+							config.PlatformAPI = "0.5"
+						})
+						it("should include processes and use the default value that is set", func() {
+							h.Mkfile(t,
+								`[[processes]]`+"\n"+
+									`type = "some-type"`+"\n"+
+									`command = "some-cmd"`+"\n"+
+									`default = true`+"\n"+
+									`[[processes]]`+"\n"+
+									`type = "web"`+"\n"+
+									`command = "other-cmd"`+"\n",
+								// default is false and therefore doesn't appear
+								filepath.Join(appDir, "launch-A-v1.toml"),
+							)
+							br, err := bpTOML.Build(lifecycle.BuildpackPlan{}, config)
+							if err != nil {
+								t.Fatalf("Unexpected error:\n%s\n", err)
+							}
+							if s := cmp.Diff(br, lifecycle.BuildResult{
+								BOM:         nil,
+								Labels:      []lifecycle.Label{},
+								MetRequires: nil,
+								Processes: []launch.Process{
+									{Type: "some-type", Command: "some-cmd", BuildpackID: "A", Default: true},
+									{Type: "web", Command: "other-cmd", BuildpackID: "A", Default: false},
+								},
+								Slices: []layers.Slice{},
+							}); s != "" {
+								t.Fatalf("Unexpected metadata:\n%s\n", s)
+							}
+						})
+					})
+
+					it("should include processes and use the default value that is set", func() {
```

Comment:
Is this a repeat of the previous context?

Reply:
It was changed as part of commit https://github.com/buildpacks/lifecycle/pull/520/commits/845f01c70137cd8ddf8c399a1b6a93f3199d6b72.